src/FavStar.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <button
+    type="button"
     class="favstar btn btn-outline-warning"
-    :class="{ active: isActive }"
+    :class="{ 'btn-sm': this.size, active: isActive }"
```

Comment:
Since we're turning a string into a boolean into a string, maybe we could just pass through the string? What's the best way to do that in Vue?